dockerfiles/header_sync/Dockerfile

```diff
@@ -2,18 +2,14 @@ FROM golang:alpine as builder
 
 RUN apk --update --no-cache add g++ git linux-headers
 
-# Build migration tool
-WORKDIR /go
-RUN GO111MODULE=auto go get -u -d github.com/pressly/goose/cmd/goose
-WORKDIR /go/src/github.com/pressly/goose/cmd/goose
-RUN GO111MODULE=auto go build -a -ldflags '-s' -tags='no_mysql no_sqlite' -o goose
-
 ENV GO111MODULE on
 
 WORKDIR /vulcanizedb
 COPY . .
 RUN go build
-
+# I think this should build the same version of goose that was already downloaded as part of go.mod,
+# but we should validate that.
+RUN go install -tags='no_mysql no_sqlite3 no_mssql no_redshift' github.com/pressly/goose/cmd/goose
```

Comment:
Seems like this would always install the latest release at https://github.com/pressly/goose - no? Or does it follow the `go.mod` because you're still in the `/vulcanizedb` `WORKDIR`?

Reply:
I believe it follows the go.mod - because the current latest release doesn't work (we're on a release candidate so if you pull the latest release you actually get an old version).

However I left the comment above to double check myself.

Reply:
**UPDATE** I'm now always getting the latest release candidate no matter what I do, which is making this difficult to verify.

Reply:
I went ahead and made these always get the @v2.7.0-rc5, which we can do now because we don't need go modules to be turned off.